### ISSUE TITLE
[!] CocoaPods could not find compatible versions for pod "react-native-plaid-link-sdk":

### ISSUE THREAD

**Issue by beqramo** [plaid/react-native-plaid-link-sdk#385]
# The problem

Briefly describe the issue you are experiencing (or the feature you want to see added to the Plaid Link 
React Native SDK). Tell us what you were trying to do and what happened instead. 

# Environment

|                        |  |
| ------------------------| ---------- |
| Plaid Link React Native          |  7.0.5               |
| ReactNative Version     |  0.64.1             |
| Occurs on Android             | no                |
| Android OS Version                    | 9.0.0           |
| Android Devices/Emulators    | Pixel 3a                |
| Occurs on iOS     | yes                |
| iOS Version           | 14.5               |
| iOS Devices/Emulators               | iPhone X                |

# Steps to Reproduce
I get this error on RN:0.64.1 on `pod install`. I tried  `pod install --repo-update`  but nothing.  I have added   `pod 'Plaid', '~> 2.1.2'` in the `podfile` too.

![image](https://user-images.githubusercontent.com/40486471/118713666-45462580-b833-11eb-9d5b-611e2e6555b0.png)

**Comment by beqramo** [comment]
solved, you need at least 11.0 ios version in pod file

**Comment by saeedtkh** [comment]
I still get this error when I do pod install. Any updates?

**Comment by dtroupe-plaid** [comment]
I don't know what version of the SDK you're using, but if it's v11 - Does your pod file specify the minimum version [is iOS 14](https://github.com/plaid/react-native-plaid-link-sdk/blob/master/example/ios/Podfile#L8)?

```
platform :ios, 14.0
```

**Comment by ZeeshanAhmadKhalil** [comment]
My iOS version is 14 in Podfile:
`platform :ios, '14.0'`

but still getting this error:
```
[!] CocoaPods could not find compatible versions for pod "Plaid":
  In snapshot (Podfile.lock):
    Plaid (= 5.6.1, ~> 5.6.1)

  In Podfile:
    Plaid (~> 5.6.1)

    react-native-plaid-link-sdk (from `../node_modules/react-native-plaid-link-sdk`) was resolved to 12.0.3, which depends on
      Plaid (~> 6.0.4)


You have either:
 * out-of-date source repos which you can update with `pod repo update` or with `pod install --repo-update`.
 * changed the constraints of dependency `Plaid` inside your development pod `react-native-plaid-link-sdk`.
   You should run `pod update Plaid` to apply changes you've made.
```

**Comment by dtroupe-plaid** [comment]
Hi @ZeeshanAhmadKhalil - I just removed those directions from the README. It's better to not specify the pod version because it's tied to the React native version of the SDK. In other words this step is unnecessary and creates this issue.
